azure/resource-containers/pf-management-group/functions/public/deploy-pfmanagementgroup.ps1

```diff
@@ -0,0 +1,25 @@
+Function Deploy-PfManagementGroup{
+    Param (
```

Comment:
This is now a function that holds the logic for whether or not we get, update, or create the resource in azure.  It is called from a deploy method in the definition class.  I couldn't find a better way of dynamically kicking off a deployment of resources from the deployment context other than to wrap a method "Deploy" in the definition class that then calls this function.  That way the deployment context only has to iterate through its arraylist of definitions and do a "$definition.deploy()"